packages/manager/modules/pci/src/components/project/images-list/images-list.controller.js

```diff
@@ -79,12 +79,16 @@ export default class ImagesListController {
     this.unavailableImages = {};
 
     forEach(this.os, (distributions, imageType) => {
```

Comment:
i think we can move it to native `Array.forEach`.

Reply:
It doesn't work with the native forEach.

Reply:
Yep, native `forEach` don't support iteration over object attributes-vallues like `Lodash.forEach`
We can do it with this `Object.entries(this.os)).forEach(([distributions, imageType]) => {})`

Reply:
I did it. I deleted lodash/forEach.